src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -670,11 +673,46 @@ public class DeleteOldRevisionsResult
 
             var deleted = DeleteRevisionsInternal(context, table, lowerIdPrefix, collectionName, changeVector, lastModifiedTicks, result.PreviousCount, revisionsToDelete, result);
 
+
+            NotifyAboutConflictRevisionsIfNeeded(conflicted, nonPersistentFlags, deleted, lowerIdPrefix);
+
             IncrementCountOfRevisions(context, lowerIdPrefix, -deleted);
             result.Remaining = result.PreviousCount - deleted;
             return result;
         }
 
+        private void NotifyAboutConflictRevisionsIfNeeded(bool conflicted, NonPersistentDocumentFlags nonPersistentFlags, long deleted, Slice lowerIdPrefix)
+        {
+            var lowerId = lowerIdPrefix.ToString();
+            lowerId = lowerId.Substring(0, lowerId.Length - 1); // cut the prefix last char (SpecialChars.RecordSeparator)
+
+            if (conflicted &&
+                nonPersistentFlags.Contain(NonPersistentDocumentFlags.ByEnforceRevisionConfiguration) == false &&
+                nonPersistentFlags.Contain(NonPersistentDocumentFlags.FromSmuggler) == false &&
+                deleted > 0)
+            {
+                var key = $"ConflictRevisionExceededMax/{lowerId}";
+
+                var config = ConflictConfiguration.Default;
+                if (config.MinimumRevisionAgeToKeep.HasValue)
+                {
+                    var alert = AlertRaised.Create(_database.Name, "Conflict Revision Exceeded to its Maximum", $"Conflict Revisions of doc \"{lowerId}\" Exceeded to its \"MinimumRevisionAgeToKeep\" ({config.MinimumRevisionsToKeep}), {deleted} revisions have been deleted.",
```

Comment:
Title: Excess number of Conflict Revisions
Message: Number of Conflict Revisions for document 'id' exceeded configured value 'value', 'deleted' revisions have been deleted.

Reply:
![RavenDB-21101 Studio](https://github.com/ravendb/ravendb/assets/48368355/acb63dc0-0870-45ad-b509-ae6abd7b8060)